twittermap/app/controllers/TwitterMapApplication.scala

```diff
@@ -21,13 +21,14 @@ class TwitterMapApplication @Inject()(val wsClient: WSClient,
   val USCityDataPath: String = config.getString("us.city.path").getOrElse("/public/data/city.sample.json")
   val cloudberryRegisterURL: String = config.getString("cloudberry.register").get
   val cloudberryWS: String = config.getString("cloudberry.ws").get
+  val twitterMapSentimentAnalysisEnabled: Boolean = config.getBoolean("twitterMap.sentimentAnalysisEnabled").get
```

Comment:
a bit too long, since this is already twitterMap, maybe we can just call it `enableSentiment`?